src/query/storage/m3/multi_fetch_result.go

```diff
@@ -22,20 +22,20 @@ package m3
 
 import (
 	"sync"
+	"time"
 
 	"github.com/m3db/m3/src/dbnode/encoding"
-	"github.com/m3db/m3/src/query/storage"
 	xerrors "github.com/m3db/m3x/errors"
 )
 
 // TODO: use a better seriesIterators merge here
 type multiFetchResult struct {
 	sync.Mutex
 	sync.Once
-	iterators        encoding.SeriesIterators
-	err              xerrors.MultiError
-	dedupeFirstAttrs storage.Attributes
-	dedupeMap        map[string]multiFetchResultSeries
+	iterators             encoding.SeriesIterators
+	err                   xerrors.MultiError
+	dedupeFirstResolution time.Duration
```

Comment:
Is there a major reason to change to `resolution time.Duration` instead of `attrs storage.Attributes`?  They are both value types that shouldn't be able to be nil'd.  

It's just I have an open PR that depends on having access to the attributes rather than just the resolution:
https://github.com/m3db/m3/pull/908